### PR TITLE
Adding ODoE documentation

### PR DESCRIPTION
**Pull request by sotorrio1**

### CODE REVIEW

**Comment by ksbeattie** [comment]
This is for #945, right?

**Comment by sotorrio1** [comment]
@ksbeattie Yes

**Comment by sotorrio1** [comment]
@mirandamartin The overview section is empty because I'm waiting on Brenda to provide that part. There is a chance she might not be able to give it to me before the release date, so we might need to submit the documentation for ODoE with just the tutorials and add the overview later.

**Review by mirandamartin — DISMISSED** [review]
Looks good!

**Comment by ksbeattie** [comment]
> @mirandamartin The overview section is empty because I'm waiting on Brenda to provide that part. There is a chance she might not be able to give it to me before the release date, so we might need to submit the documentation for ODoE with just the tutorials and add the overview later.

I think that rather than have an empty page, it would be better to either fill it in with whatever brief text you can come up with, or just leave it out entirely.  Then update it later either way.

**Comment by ksbeattie** [comment]
You're gonna hate me, but might the screen shots be made easier to read?  Perhaps it's just my old eyes, but I think for several of these, if the window was made smaller when grabbing the screenshot, the resulting text on the screen would easier to read.

**Comment by sotorrio1** [comment]
> > @mirandamartin The overview section is empty because I'm waiting on Brenda to provide that part. There is a chance she might not be able to give it to me before the release date, so we might need to submit the documentation for ODoE with just the tutorials and add the overview later.
> 
> I think that rather than have an empty page, it would be better to either fill it in with whatever brief text you can come up with, or just leave it out entirely. Then update it later either way.

I already talked to Brenda and we'll add text to the section.

**Comment by sotorrio1** [comment]
> You're gonna hate me, but might the screen shots be made easier to read? Perhaps it's just my old eyes, but I think for several of these, if the window was made smaller when grabbing the screenshot, the resulting text on the screen would easier to read.

haha no hate, I haven't even checked the screenshots. I can re-take them so it's easier to read.

File: docs/source/chapt_odoe/tutorial-createcand.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+#. Start FOQUS and click the ‘SDoE’ icon.
+
+   .. figure:: figs/1_SDoE_main.png
+      :alt: SDoE Main Window
+      :name: fig.SDoE_main2
+
+      SDoE Main Window
+
+
+#. Select the radio button **Robust omptimality-based DoE (ODoE)**
```
Comment (lbianchi-lbl) [inline docs/source/chapt_odoe/tutorial-createcand.rst:15]:
```suggestion
#. Select the radio button **Robust optimality-based DoE (ODoE)**
```

File: docs/source/chapt_odoe/tutorial-loadcand.rst [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+#. Start FOQUS and click the ‘SDoE’ icon.
+
+   .. figure:: figs/1_SDoE_main.png
+      :alt: SDoE Main Window
+      :name: fig.SDoE_main1
+
+      SDoE Main Window
+
+
+#. Select the radio button **Robust omptimality-based DoE (ODoE)**
```
Comment (lbianchi-lbl) [inline docs/source/chapt_odoe/tutorial-loadcand.rst:15]:
```suggestion
#. Select the radio button **Robust optimality-based DoE (ODoE)**
```

**Review by mirandamartin — APPROVED** [review]
Looks good

**Pull request merged**